code/game/objects/items/shields.dm

```diff
@@ -355,3 +355,44 @@
 		slot_flags = null
 		to_chat(user, "<span class='notice'>[src] can now be concealed.</span>")
 	add_fingerprint(user)
+
+/obj/item/shield/riot/briefcase_shield //this is for a traitor item which is why the name and description is the same as a regular briefcase
+	name = "briefcase"
+	desc = "It's made of AUTHENTIC faux-leather and has a price-tag still attached. Its owner must be a real professional."
+	icon_state = "caseshield0"
+	lefthand_file = 'icons/mob/inhands/equipment/shields_lefthand.dmi'
+	righthand_file = 'icons/mob/inhands/equipment/shields_righthand.dmi'
+	max_integrity = 100
+	transparent = FALSE
+	slot_flags = null
+	force = 3
+	throwforce = 3
+	throw_speed = 3
+	throw_range = 4
+	w_class = WEIGHT_CLASS_BULKY
+	var/active = 0
+
+/obj/item/shield/riot/briefcase_shield/hit_reaction(mob/living/carbon/human/owner, atom/movable/hitby, attack_text = "the attack", final_block_chance = 0, damage = 0, attack_type = MELEE_ATTACK)
+	if(active)
+		return ..()
+	return 0
+
+/obj/item/shield/riot/briefcase_shield/attack_self(mob/living/user)
```

Comment:
Needs a cooldown.

Reply:
I thought it'd be alright because the teleshield I based it off doesn't have one, but if you want me to add one I will when  I get back home

Reply:
Meh fair, it's already a bit pricey.

Reply:
about the price, I think i'm gonna ask on PR discussion on the discord about it cause from what i've heard  so far from people it's a bit high for what it is